inlong-manager/manager-dao/src/main/resources/mappers/StreamSourceEntityMapper.xml

```diff
@@ -342,6 +342,29 @@
             </if>
         </where>
     </select>
+    <select id="selectNeedUpdateByAgentIpAndCluster" resultType="java.lang.Integer">
```

Comment:
Change the name to `selectHeartbeatTimeoutIds`, and no need transfer the `statusList` param.

Reply:
done